LocalChat/LocalChat/Controllers/SettingViewController.swift

```diff
@@ -119,36 +122,66 @@ class SettingViewController: UIViewController, UITableViewDelegate, UITableViewD
     
     func showAlert() {
         checkedNumber = 0
-        let alert = UIAlertController(title: "Naaaaa!", message: "Please select at least two user", preferredStyle: UIAlertControllerStyle.Alert)
+        let alert = UIAlertController(title: Constants.alertMessages.titleForWarning, message: Constants.alertMessages.selectUserMessage, preferredStyle: UIAlertControllerStyle.Alert)
         alert.addAction(UIAlertAction(title: "OK", style: UIAlertActionStyle.Default, handler: {action in self.viewWillAppear(true)}))
         presentViewController(alert, animated: true, completion: nil)
```

Comment:
Though only one line to execute in the handler put this "self.viewWillAppear(true)" in separate line for better understanding